player.gd

```diff
@@ -191,7 +207,8 @@ func _rollback_tick(delta, _tick, _is_fresh):
 				velocity.y += gravity * delta
 			else:
 				velocity.y = 200
-			velocity.x = move_toward(velocity.x, input.direction.x * air_max_speed, air_acceleration)
+			if move_walljump:
+				velocity.x = move_toward(velocity.x, input.direction.x * air_max_speed, air_acceleration)
```

Comment:
I thought if we are enabling move_walljump, then there should be a piece of code that executes a walljump when pressing a movement key, but I can't seem to find it.